ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/event-handlers.cfg

```diff
@@ -28,4 +28,16 @@ event-handler trigger-on-boot
    trigger on-boot
    action bash echo "on-boot"
 !
+event-handler trigger-on-maintenance
+   trigger on-maintenance enter interface Ethernet4 all
+   action bash echo "on-maintenance"
+!
+event-handler trigger-on-maintenance1
+   trigger on-maintenance enter bgp 10.0.0.2 after linkdown
```

Comment:
this is not working:
```
wan1-dc1(config-handler-Test1)#trigger on-maintenance enter bgp 10.0.0.1 after linkdown 
% Invalid input
```

you need stage before that
```
wan1-dc1(config-handler-Test1)#trigger on-maintenance enter bgp 10.0.0.1 after stage ?
  bgp       Maintenance stage BGP where BGP maintenance profile is applied
  linkdown  Maintenance stage linkdown where MLAG port-channels are shutdown
  mlag      Maintenance stage MLAG where MLAG forceful failover is done
  ratemon   Maintenance stage ratemon where interface maintenance profile is applied
  ```

Reply:
Fixed